features/support/env.rb

```diff
@@ -98,7 +98,7 @@ def dm_pre_load_wait_time
 
 Capybara.asset_host = dm_frontend_domain
 Capybara.save_path = "reports/"
-Capybara.default_max_wait_time = 0.05
+Capybara.default_max_wait_time = 5
```

Comment:
It's not clear from the commit message why the timeout needs to be increased.

Reply:
Crossed in the wires again I think
https://github.com/alphagov/digitalmarketplace-functional-tests/compare/af20722efb4c78bf2a9049349f8e1596faf7829c..319557b5398c648b5cd779189cbd4355f89b9c79

Reply:
5 seconds is a very long time for this, and could easily lead to extremely long test runs.